src/components/reusables/__tests__/Fieldset-test.tsx

```diff
@@ -0,0 +1,41 @@
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import Fieldset from "../Fieldset";
+
+describe("Fieldset", () => {
+  let wrapper: Enzyme.ShallowWrapper<any, {}>;
+  let legend = "TEST_LEGEND";
+  let input1 = <input value="A" />;
+  let input2 = <input value="B" />;
+  beforeEach(() => {
+    wrapper = Enzyme.shallow(
+      <Fieldset legend={legend} elements={[input1, input2]} />
+    );
+  });
+  it("displays a legend", () => {
+    let legend = wrapper.find("legend");
+    expect(legend.length).to.equal(1);
+    expect(legend.text()).to.equal("TEST_LEGEND");
+  })
+  it("displays elements", () => {
+    let inputs = wrapper.find("input");
+    expect(inputs.length).to.equal(2);
+    expect(inputs.at(0).prop("value")).to.equal("A");
+    expect(inputs.at(1).prop("value")).to.equal("B");
+  });
+  it("optionally displays a label", () => {
```

Comment:
This is confusing because you can "optionally" display a label but you are always displaying a legend. The label is just extra text at the end of the legend. I don't think the names match up semantically.